src/otx/core/model/classification.py

```diff
@@ -60,10 +61,31 @@ def __init__(
             scheduler=scheduler,
             metric=metric,
             torch_compile=torch_compile,
-            train_type=train_type,
         )
         self.input_size: tuple[int, int]
 
+    def _create_model(self) -> nn.Module:
```

Comment:
Where was this method before and why do we need to replicate it 3 times now?

Reply:
this method was in each specific model implementation. Actually, there was a code duplication, that's why I put it in the base class.

Reply:
_build_model is called with different classes to identify classification layers. Not sure if it is required. Need to revisit it. For now, I just copied it.